net.krafting.PleasureDVR.yml

```diff
@@ -0,0 +1,31 @@
+app-id: net.krafting.PleasureDVR
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: PleasureDVR.py
+finish-args:
+  -  --share=ipc
+  -  --device=dri
+  -  --share=network
+  -  --socket=wayland
+  -  --socket=fallback-x11
+  -  --filesystem=xdg-videos
+
+modules:
+  - name: PleasureDVR
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 PleasureDVR.py /app/bin/PleasureDVR.py
+      - install -Dm 0644 net.krafting.PleasureDVR.desktop  /app/share/applications/net.krafting.PleasureDVR.desktop
+      - install -Dm 0644 net.krafting.PleasureDVR.metainfo.xml  /app/share/metainfo/net.krafting.PleasureDVR.metainfo.xml
+      - install -Dm 0644 images/pleasuredvr-512x512.png /app/share/icons/hicolor/256x256/apps/net.krafting.PleasureDVR.png
+    sources:
+      - type: git
+        url: https://gitlab.com/Krafting/pleasure-dvr-gtk
+        tag: beta-1.6
+
+  - chaturbate-dvr.json
```

Comment:
if this is not auto generated, there is no reason to have it separate.

Reply:
This is generated automatically and I just rename the file and put them here. thanks to:

`go run github.com/dennwc/flatpak-go-mod@latest ./`

This generate a go.mod.yml and modules.txt